app/_locales/en/messages.json

```diff
@@ -259,6 +259,18 @@
   "betaWelcome": {
     "message": "Welcome to MetaMask Beta"
   },
+  "blockExplorerAccountAction": {
+    "message": "Account"
```

Comment:
Ideally these would have descriptions that link them back up to the `viewOnCustomBlockExplorer` and `viewOnEtherscan` messages, so that translators know which context these words will be used in.